golang/internal/lab4/lab4.go

```diff
@@ -0,0 +1,29 @@
+package lab4
+
+import "math"
+
+func ToRadians(deg float64) float64 {
+	return (deg) * (math.Pi / 180.0)
+}
+
+func Calculate(x float64, b float64) float64 {
+	return (1 + math.Pow(math.Sin(ToRadians(math.Pow(b, 3)+math.Pow(x, 3))), 2)) / math.Cbrt(math.Pow(b, 3)+math.Pow(x, 3))
+}
+
+func TaskA(begX, finX, detX, b float64) ([]float64, []float64) {
+	var yList = make([]float64, 0)
+	var corrXList = make([]float64, 0)
```

Comment:
вот тут вы могли бы прикинуть размер 
```suggestion
	var yList = make([]float64, 0, (finX-begX)/detX) <- и округлить в большую сторону или просто +1
	var corrXList = make([]float64, 0)
```